unlock-app/src/pages/_app.tsx

```diff
@@ -18,8 +18,8 @@ const UnlockApp = ({ Component, pageProps }: AppProps) => {
   }, [])
 
   return (
-    <GlobalWrapper pageProps={pageProps}>
```

Comment:
I'm a bit worried about this. Have you checked if we pass any props to page aside from the checkout page and confirmed there is nothing dependent on it.

Reply:
is not used in other parts, I also checked the previous PR to see when this was introduced.

ref:
[7929](https://github.com/unlock-protocol/unlock/pull/7929)
[8010](https://github.com/unlock-protocol/unlock/pull/8010)